src/plugins/muteNewGuild.ts

```diff
@@ -0,0 +1,19 @@
+import definePlugin from "../utils/types";
+
+export default definePlugin({
+    name: "MuteNewGuild",
+    description: "Mutes newly joined guilds",
+    authors: [{
+        name:"Glitchy",
+        id: 269567451199569920n
+    }],
+    patches: [
+        {
+            find: "acceptInvite:function",
```

Comment:
there are two modules containing this string
```suggestion
            find: ",acceptInvite:function",
```